gensim/matutils.py

```diff
@@ -1328,10 +1328,23 @@ class MmReader(object):
     This allows us to process corpora which are larger than the available RAM.
 
     """
+    @contextmanager
+    def open_file(self, input):
+        # Generates 'with' like behaviour excepting closing the file object
+        mgr = utils.file_or_filename(self.input)
+        exc = False
+        try:
+            yield mgr
+        except StandardError:
```

Comment:
please add the comment, what's a reason for exception here

Reply:
if handle any - why not `Execption` instead of `StandartError` ?

Reply:
Hmm , ya my bad . I will use except Exception.